lib/compiler/passes/generate-bytecode.js

```diff
@@ -254,6 +259,7 @@ function generateBytecode(ast) {
       }),
       [op.SILENT_FAILS_OFF],
       buildCondition(
+        negative ? -match : match,
```

Comment:
really?

Reply:
> When you learn about what the match means (and I add an explanation to this pass), your question will gone, I guess. 

You're the guy who made me explain how typescript worked.  Now you won't explain your own code 😂

Reply:
I do not see what lead you to that assumption in my phrase, which says the exact opposite. If my language not so good (I am ready to admit it, I'm not a native speaker), let me known in a straightforward way, and I'll try to use other words